templates/adherent/profile.html.twig

```diff
@@ -10,6 +16,10 @@
         Kernel.onLoad(function() {
             {{ js.address(form.address, { cityAutocomplete: true }) }}
         });
+
+        $(document).ready(function() {
+            $('.select2').select2();
```

Comment:
Can you rename this class to have more specific name please ?

Reply:
:heavy_check_mark: